### ISSUE TITLE
How to implement tests?

### ISSUE THREAD

**Issue by tischi** [image-transform-converters/image-transform-converters#3]
We really need tests to ensure that the converters are doing the right thing!
How to do this?

**Comment by bogovicj** [comment]
Agreed.

It won't guarantee correctness, but something like
```
A a1;
b1 = convert_a_to_b( a1 )
a2 = convert_b_to_a( b1 );
b2 = convert_b_to_a( a2 );
assertEquals( a1, a2 );
assertEquals( b1, b2 );
```
would go a long way (assuming the a's and b's are not empty / trivial )

**Comment by tischi** [comment]
Yes. In fact that was also what I was thinking, convert in a circle and check that you get the same back as you started with. We could try to find all possible paths ending at the starting point automatically using @NicoKiaru's tricks :-)

**Comment by bogovicj** [comment]
Implemented this idea - it was a bit uglier than I expected, because the default `equals` method for `AffineTransform3D` is wrong.

See https://github.com/image-transform-converters/image-transform-converters/commit/6dfc10f369c697ead4d95bf693eff65e6739c65c
